packages/secrets/build.js

```diff
@@ -0,0 +1,14 @@
+import esbuild from 'esbuild';
+
+esbuild
```

Comment:
Config is very similar to the one in the cli app that's part of this changeset. It's probably fine for now, but if we start to see patterns of this, we might want to give some thought for having a default, extendable config (a la eslint).